gw_eccentricity/load_data.py

```diff
@@ -526,6 +535,14 @@ def get_defaults_for_nr():
     extrap_order: int
         Extrapolation order to use for loading the waveform data.
         NOTE: This is used only for sxs catalog formatted waveforms.
+
+    keep_memory: bool
+        If False, remove memory contribution from the waveform modes.  This will
+        require metadata file to find t_relax which is used to start the
+        integration for computing memory contribution.  NOTE: This can be used
+        only in the newer sxs catalog formatted waveforms with
+        origin=`SXSCatalog`
```

Comment:
Does the code raise a helpful Exception if this is used with catalogs other than SXSCatalog?

Reply:
It should by design since it is included only in the list of kwargs for SXSCatalog. I will share an example case.

Reply:
Here is the error message if `keep_memory` is used, for example, with `SXSCatalog_old`
![image](https://github.com/user-attachments/assets/824a48ab-4207-4ae5-8b44-742f1d1b63de)